### PR TITLE
include uMatriks

### PR DESCRIPTION
**Pull request by peter-scholtens**
A client Ubuntu Touch, or UBports.

### CODE REVIEW

**Comment by t3chguy** [comment]
Maturity doesn't follow the convention, and the instructions seem a little vague type it in where
At least the other ones give it as a command that an be typed into the main input of the app (chat input)

File: src/components/HomePage.js [hunk 1]
```diff
@@ -120,6 +120,15 @@ var unlinkable_clients = [
         maturity: "Alpha",
         comments: "Access any room anywhere in Matrix via good old IRC!",
     },
+    {
+        name: "uMatriks",
+        logo: "img/uMatriks-256px.png",
+        author: "LarreaMikel",
+        homepage: "https://github.com/LarreaMikel/uMatriks",
+        maturity: "Alpha",
+        room_instructions(userId)  { return <span>Click "Join room"</span> },
```
Comment (t3chguy) [inline src/components/HomePage.js]: the fn signature should be `room_instructions(alias)` but what I meant is that the instruction seems really useless, doesn't even mention the alias to join, look at the instruction for PTO for instance `room_instructions(alias)  { return <span>Type <code>/join <b>{ alias }</b></code></span> },`

**Comment by t3chguy** [comment]
The whole point of matrix.to is to instruct users on how to join a room/invite a user in a specific client, I know nothing about uMatriks but look at the other clients, they all list instructions containing the room_id/user_id respectively: https://github.com/matrix-org/matrix.to/pull/34/files#diff-e811385e94cd1548745951759faedd55R71

**Comment by t3chguy** [comment]
Looks much better :D

**Comment by bwindels** [comment]
Closing as the api to add a client has changed. Please resubmit if still interested.

**Pull request closed without merging**